x-pack/plugins/security_solution/server/lib/detection_engine/signals/filter_events_with_list.ts

```diff
@@ -37,11 +38,28 @@ export const filterEventsAgainstList = async ({
       return eventSearchResult;
     }
 
+    const exceptionItemsWithLargeValueLists = exceptionsList.reduce<ExceptionListItemSchema[]>(
```

Comment:
NOTE: I found that when an exception item did not include a large value list, it still entered the logic of mapping through the exceptions to find large value lists and resulted in `filteredHitsPromises` being `[[]]`. Since it read it as not null, and having a length, we would pass back a `SignalSearchResponse` with 0 hits and this was passed to the function creating the signals.